nixos/modules/services/hardware/trezord.nix

```diff
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.trezord;
+in {
+  
+  ### interface
+
+  options = {
+    services.trezord = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable Trezor bridge daemon, for use with Trezor hardware bitcoin wallets.
+        '';
+      };
+
+      user = mkOption {
```

Comment:
To my mind, adding options "just because" is not real flexibility, but rather bloats the configuration state space to no benefit. Absent a plausible use case for running a daemon under a different user, please re-consider whether it needs to be possible to do so. This holds doubly true for daemons that can work with dynamic uids, as down the line we can rely on systemd to do that for us, obviating the need to declare daemon users up-front for those services.